langwatch/src/server/tracer/types.ts

```diff
@@ -247,12 +247,14 @@ export type ElasticSearchSpan = Omit<
 };
 
 export type TraceInput = {
-  value: string;
+  // We hope this is a string, but it could be anything!
+  value: string | object;
   satisfaction_score?: number;
 };
 
 export type TraceOutput = {
-  value: string;
+  // We hope this is a string, but it could be anything!
+  value: string | object;
```

Comment:
actually it shouldn't be anything, at least not for the ElasticSearchTrace, I'm afraid if we change it like this we might inadvertdly save the trace in ES without first json strigifying it and that would be worse, before merging this let me rethink how to handle it